airgun/entities/host.py

```diff
@@ -331,8 +329,9 @@ def step(self, *args, **kwargs):
         action_name = kwargs.get('action_name')
         self.VIEW = self.ACTIONS_VIEWS.get(action_name)
         if not self.VIEW:
-            raise ValueError('Please provide a valid action name.'
-                             ' action_name: "{0}" not found.')
+            raise ValueError(
+                'Please provide a valid action name.' ' action_name: "{0}" not found.'
```

Comment:
i wonder if that 0 index needs to be there

Reply:
Good call - I think this is one of the situations where order of pyupgrade and black in pre-commit may have an impact.